domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/service/casetype/CaseTypeServiceImpl.java

```diff
@@ -52,9 +53,7 @@ public CaseTypeServiceImpl(CaseTypeRepository repository,
         this.metadataFieldService = metadataFieldService;
     }
 
-    @Override
-    public void createAll(JurisdictionEntity jurisdiction, Collection<CaseTypeEntity> caseTypes) {
-
+    private void validate(JurisdictionEntity jurisdiction, Collection<CaseTypeEntity> caseTypes, Set<String> missingUserRoles) {
```

Comment:
can you move the private methods after the public methods plesae

Reply:
done